packages/doc/src/components/Layout/Layout.jsx

```diff
@@ -131,9 +131,7 @@ const Layout = ({
       <MainWrapper>
         <Grid>
           <Header showMenu={showMenu} toggleMenu={toggleMenu}>
-            <HeaderLink to="/guidelines/product-content/introduction">
-              Guidelines
-            </HeaderLink>
+            <HeaderLink to="/guidelines/tokens/borders">Guidelines</HeaderLink>
```

Comment:
Why click on the `Guidelines` should lead the user to `/guidelines/tokens/borders` and not `/guidelines`?

Reply:
If you keep only on the `/guidelines` route, you will be redirected to an empty screen.

Reply:
Example:
![image](https://user-images.githubusercontent.com/46993493/165784948-dbecdd1d-a649-432f-a825-9c5548ae2018.png)